aiosonic/connection.py

```diff
@@ -31,10 +33,17 @@ def __init__(self, connector: TCPConnector) -> None:
         self.blocked = False
         self.temp_key: Optional[str] = None
         self.requests_count = 0
+        self.cycled = False
```

Comment:
Can you explain a bit this variable?

I think I need to better document this class which is a bit complex for the nature of the problem :smile:

Reply:
It's a safe-guard against putting the same Connection object into the pool twice. I can test without it and see if it makes a difference.